compiler/compiler.go

```diff
@@ -491,19 +491,25 @@ func (c *compiler) MemberNode(node *ast.MemberNode) {
 		}
 	}
 
-	c.compile(base)
-	if node.Optional {
-		ph := c.emit(OpJumpIfNil, placeholder)
-		c.chains[len(c.chains)-1] = append(c.chains[len(c.chains)-1], ph)
-	}
-
-	if op == OpFetch {
+	// Implementation of direct env access
+	if ident, ok := base.(*ast.IdentifierNode); ok && ident.Value == "env" {
```

Comment:
What about implementing this in the IdentifierNode directly? Let's add the opcode `OpLoadEnv` which will put the env on the stack and then use standard MemberNode implementation. 

Right now next expr features will not work:

```
env.FooBar
env?.optionalValue
```